3lvia/kunde-elvia-web
# feat(poweroutage + ui): suuport for changing bounds to selected power outage

OMEGA-175

## What is the reason for this change?

To be able to zoom in on focused power outage and zoom out to original when not focusing on power outage

### What is the expected result?

Change bounds to focused power outage if focused, and original when losing focus

## How is it solved?

Change bounds to focused power outage, or to original bounds, if not focused

![focus-power-outage](https://user-images.githubusercontent.com/10927589/94928862-941d9580-04c4-11eb-8966-352f9f66da2b.gif)

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

- Click on poweroutage, should change bounds to poweroutage
- Close power outage, should change bounds to show all power outages
